tools/waveica/waveica.xml

```diff
@@ -61,10 +64,12 @@
                 <expand macro="input_data"/>
             </when>
             <when value="2">
+                <expand macro="input_data"/>
                 <section name="input_metadata" title="Input metadata table" expanded="true">
                     <param name="metadata" label="Input metadata" type="data" format="csv,tsv,parquet" help="" />
-                </section> 
-                <expand macro="input_data"/>
+                </section>
+                <param name = "transpose_feature_table" label="Transpose feature table" type="boolean" checked="false" 
+                truevalue="TRUE" falsevalue="FALSE" help="Transpose if feature names are -> column 1 and sampleNames are column" />
```

Comment:
Somehow the `help` doesn't help.

Reply:
True. This could be improved a bit :D @zargham-ahmad

Reply:
Updated in [288aaae](https://github.com/RECETOX/galaxytools/pull/378/commits/288aaae211afc339670bc205845a4393665a8455)

Reply:
Is it better now? or maybe it seems more complicated now :D

Reply:
Yeah, now it's actually a sentence :D but still takes a minute to understand. I'm thinking of it's even needed to explain, I suppose "transpose" is pretty clear. Anyway, maybe a more straightforward formulation: "Swap sample names with feature names as column headers (to fit aplcms outputs)."

Reply:
Yes, this looks good as well